src/app/pages/dashboard/dashboard.page.ts

```diff
@@ -12,7 +12,8 @@ import { Router, ActivatedRoute} from "@angular/router"
 
 import { Apollo } from "apollo-angular";
 import gql from "graphql-tag";
-import html2canvas from 'html2canvas';
+// import html2canvas from 'html2canvas';
```

Comment:
Is this really needed for the logout functionality fix? 

Seems extraneous to include this in the PR